pkg/rslog/logger_impl.go

```diff
@@ -174,7 +174,6 @@ func (l logrusEntryWrapper) Logf(msg string, args ...interface{}) {
 
 var defaultLogger Logger
 var once = &sync.Once{}
-var mutex sync.RWMutex
```

Comment:
Would it make more sense to rename the mutex in `debug.go` to `debugMutex` so we can still keep them separate?

Reply:
ah, yes - that does make sense to me. Good call.